nofos/nofos/templates/nofos/nofo_edit.html

```diff
@@ -58,144 +58,254 @@ <h1 class="font-heading- margin-0" title="{{ nofo_name_str }}">{{ nofo_name_str
   </span>
 </div>
 
-{% if nofo.status == 'cancelled' %}
+<!-- Consolidated NOFO Status and Actions Summary Box -->
 <div class="usa-summary-box" role="region" aria-labelledby="summary-box-key-information">
   <div class="usa-summary-box__body">
-    <h2 class="usa-summary-box__heading font-heading-sm" id="summary-box-key-information">
-      NOFO is ‘<a href="{% url 'nofos:nofo_edit_status' nofo.id %}">{{ nofo.get_status_display }}</a>’.
-    </h2>
-    <div class="usa-summary-box__text">
-      <p>Cancelled NOFOs exist as a record for posterity but can’t be edited in any way. If you would like to restore this NOFO, <a href="{% url 'nofos:nofo_edit_status' nofo.id %}">change its status</a> to something else.</p>
-    </div>
-  </div>
-</div>
-{% elif nofo.archived %}
-<div class="usa-summary-box" role="region" aria-labelledby="summary-box-key-information">
-  <div class="usa-summary-box__body">
-    <h2 class="usa-summary-box__heading font-heading-sm" id="summary-box-key-information">
-      NOFO is ‘Archived’.
-    </h2>
-    <div class="usa-summary-box__text">
-      <p>Archived NOFOs exist as a record for posterity but can’t be edited in any way. If you would like to restore this NOFO, please get in touch with Paul or Adam.</p>
-      {% if nofo.successor %}
-        <p>This is a past version of <a href="{% url 'nofos:nofo_edit' nofo.successor.id %}">{{ nofo.successor }}</a></p>
+    <h4 class="usa-summary-box__heading font-heading-sm" id="summary-box-key-information">
+      <!-- Status Dropdown Form -->
+      {% if not nofo.archived %}
+        <form id="nofo-status--form" method="post" action="{% url 'nofos:nofo_edit_status' nofo.id %}">
+          {% csrf_token %}
+          <div class="usa-form-group margin-bottom-3 margin-top-0">
+              <label class="usa-label display-inline-block font-sans-md margin-top-0" for="{{ status_form.status.name }}"><strong>Status:</strong></label>
+
+            <select
+              class="usa-select border-2px width-auto display-inline-block margin-top-0"
+              name="{{ status_form.status.name }}"
+              id="{{ status_form.status.name }}"
+              onchange="this.form.submit()"
+            >
+              {% for value, label in status_form.status.field.choices %}
+                {% if value == '' %}
+                  <option disabled>{{ label }}</option>
+                {% else %}
+                  <option value="{{ value }}" {% if status_form.status.value == value %}selected{% endif %}>
+                    {{ label }}
+                  </option>
+                {% endif %}
+              {% endfor %}
+            </select>
+          </div>
+        </form>
+      {% else %}
+          NOFO is ‘Archived’.
```

Comment:
Can we remove the single quotes? Now we don't have this for any other status, let's just ditch them.